content/how-to-todo-in-ember-template-lint.md

```diff
@@ -0,0 +1,121 @@
+---
+title: How To Todo In Ember Template Lint
+authors:
+  - melanie-sumner
+date: 2021-02-02T00:00:00.000Z
+tags:
+  - addons
+  - '2021'
+---
+
+## Introduction
+
+**At the time of this writing, this feature is available in `ember-template-lint@3.0.0-beta.3`.**
+
+The new `todo` feature in the `ember-template-lint` addon provides a new option to the existing states for linting rules.
+
+With this new functionality, linting issues that are found can be converted into a `todo`, giving developers the ability to strategically plan the work to resolve issues in existing code. A `todo`, can be set to automatically become a `warning` and then an `error` with the use of a supporting feature called decay days.
+
+<img src="/images/blog/how-to-todo-in-ember-template-lint/todo-warning-error-graphic.png" alt="todo, warning, error">
+
+Previously, running the  `lint:hbs` script (default in an Ember app) would cause `ember-template-lint` to run on all of the `.hbs` files, and return a result that included number of `warnings` and number of `errors` found.
+<img src="/images/blog/how-to-todo-in-ember-template-lint/terminal-without-todo.png" alt="a terminal showing results that include 2 errors and 0 warnings" />
+
+Now, however, we have a new option- we can also include number of `todos`.  After updating the  `lint:hbs` script in the `package.json` file to use the `--include-todo` flag (we want to be reminded that we have todos!), like this:
```

Comment:
I recommend using a dash instead of a hyphen.

```suggestion
Now, however, we have a new option—we can also include number of `todos`. First, update the  `lint:hbs` script in the `package.json` file to use the `--include-todo` flag (we want to be reminded that we have todos!):
```